src/components/container/DailyDataGoal/DailyDataGoal.stories.tsx

```diff
@@ -114,34 +115,40 @@ export const SleepGoalLive = {
     render: render
 };
 
-export const WearTimeGoalLive = {
+const renderWithDRC = (args: DailyDataGoalProps) => <Layout colorScheme='auto'>
+    <DateRangeCoordinator intervalType="Day">
+        <DailyDataGoal {...args} />
+    </DateRangeCoordinator>
+</Layout>;
+
+export const LiveGarminSleepScore = {
```

Comment:
Minor suggestion - I think the story names are a little confusing here.  It sounds like the only difference between the story you renamed and the story you edited is the type of data, but one also uses the `DateRangeCoordinator` and the other doesn't.  My suggestion would actually be 4 stories in this case; two for each data type with and without the `DateRangeCoordinator`.

Reply:
There is a unified sleep dailyDataType, but in this story it wasn't used. I think it makes more sense to update to use the unified vs manf specific sleep data type and revert the name back to imply it is not manf specific data. 

I'll update the new story I added with the Date Range Coordinator to use the same unified data type. There are 65ish DailyDataTypes. Adding a live story for each one is out of scope for this change.